samples/widgets/showextent/manifest.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "showextent",
-  "label": "Show Extent",
```

Comment:
The label is required.
The label for default locale (en) is read from here, other locales are from translations.

Reply:
I added the label back in.